### PR TITLE
Simplify data a bit

### PR DESCRIPTION
**Pull request by xtuc**
This introduce the notion of id, it removes some info that we can generate and will probably help for automation in the future.

There's only one proposal that has no website, so the link is broken. I think that's not a big deal?

### CODE REVIEW

**Comment by ljharb** [comment]
Which?

**Review by jasonwilliams — APPROVED** [review]

**Comment by jasonwilliams** [comment]
@xtuc which proposal has no website?

**Comment by codehag** [comment]
@xtuc ping! do you know which proposals have no website?

**Comment by xtuc** [comment]
It's https://github.com/tc39/proposal-regexp-legacy-features. https://tc39.github.io/proposal-regexp-legacy-features is 404

**Comment by codehag** [comment]
Can we have a way of overriding the default link? And for that repo make the link https://github.com/tc39/proposal-regexp-legacy-features?

**Comment by serspump4299** [comment]
Like

**Comment by codehag** [comment]
Once the other data changes merge, i will add this in as well

File: _includes/proposals.html [hunk 1]
```diff
@@ -33,11 +31,9 @@ <h4 class="featurelist__item__title flex-grow">
                   </li>
                 {% endfor %}
                 {% endif %}
-                {% if proposal.specification %}
-                <li class="featurelist__item__spec">
-                  <a href="{{ proposal.specification }}">Specification</a>
-                </li>
-                {% endif %}
+              <li class="featurelist__item__spec">
+                <a href="https://tc39.github.io/{{ proposal.specification }}">Specification</a>
```
Comment (codehag) [inline _includes/proposals.html]: made a mistake during the rebase

```suggestion
                <a href="https://tc39.github.io/{{ proposal.id }}">Specification</a>
```

**Comment by codehag** [comment]
@xtuc does this look ok to you?

**Comment by codehag** [comment]
@xtuc can you take a look? I would like to merge once i have your ok.

File: _data/stage3.yml [hunk 2]
```diff
@@ -1,27 +1,27 @@
 - title: globalThis
+  id: proposal-global
+  specification: true
```
Comment (xtuc) [inline _data/stage3.yml]: maybe we could rename `specification` to `has_specification`?

**Review by xtuc — COMMENTED** [review]
otherwise LGTM

**Pull request merged**